internal/services/containers/kubernetes_cluster_resource_test.go

```diff
@@ -36,6 +37,37 @@ func TestAccKubernetesCluster_hostEncryption(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesCluster_defaultNodePool(t *testing.T) {
```

Comment:
TestAccKubernetesCluster_nodeLabels

And add a similar test for node pool resource.